docs/reference/seismometer.rst

```diff
@@ -38,3 +38,20 @@ Public API
    show_info
    show_cohort_summaries
    target_feature_summary
+
+
+.. _custom-visualization-controls:
+
+Custom Visualization Controls
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. currentmodule:: seismometer.controls
+
+.. autosummary::
+   :toctree: api/
+
+
+   explore.ExplorationWidget
+   explore.ExplorationModelSubgroupEvaluationWidget
+   explore.ExplorationCohortSubclassEvaluationWidget
+   explore.ExplorationCohortOutcomeInterventionEvaluationWidget
```

Comment:
These both have the same description. Can you update the intervention eval version to include the word intervention?

Reply:
FINC